pocs/utils/images/__init__.py

```diff
@@ -101,19 +101,11 @@ def _make_pretty_from_fits(fname, **kwargs):
     norm = ImageNormalize(interval=PercentileInterval(percent_value), stretch=LogStretch())
     wcs = WCS(fname)
 
-    nxpix = header.get('NAXIS1')
-    nypix = header.get('NAXIS2')
-
-    dpi = 160
-
-    fnx_size = nxpix / dpi
-    fny_size = nypix / dpi
-
-    plt.figure(figsize=(fnx_size, fny_size), dpi=dpi)
+    plt.figure(figsize=(10, 8), dpi=75)
```

Comment:
`dpi` a bit low? This will give a 750 x 600 pixel image.  I'd be tempted to 100-150 dpi.